src/shell/element/stack.rs

```diff
@@ -861,7 +861,11 @@ impl Program for CosmicStackInternal {
                 theme::Container::custom(|theme| iced_widget::container::Appearance {
                     icon_color: Some(Color::from(theme.cosmic().background.on)),
                     text_color: Some(Color::from(theme.cosmic().background.on)),
-                    background: Some(Background::Color(theme.cosmic().palette.neutral_3.into())),
+                    background: Some(Background::Color(if theme.cosmic().is_dark {
+                        Color::from_rgba(0.149, 0.149, 0.149, 1.0)
```

Comment:
Aren't these defined in the theme? If not, can we make these at least constants please, because I feel like we are using these in multiple places.

Additionally, we are hard-coding specific values for light and dark here. Can't we take the user-selected background color?

Reply:
In our figma designs, only the text color and borders inherit from the system theme.
I can make constants for them. Though I'm currently working on hiding the text overlay when text is not overflowing.

Reply:
> I can make constants for them.

Thanks

> Though I'm currently working on hiding the text overlay when text is not overflowing.

Estimating the size of the widgets beforehand was quite complex for the various design. Any changes making this simpler are very welcome.